docs/headless-cms/09-02-asyncapi-service.md

```diff
@@ -0,0 +1,13 @@
+---
+title: CMS AsyncAPI Service
+type: Metrics
+---
+
+This table shows the CMS AsyncAPI Service custom matrics, their descriptions, and types.
+
+| Name | Type | Description |
+|------|-------------|------|
+| `cms_services_http_request_and_mutation_duration_seconds` | histogram | Specifies a number of assets that the service received for processing and mutated within a given time series. |
+| `cms_services_http_request_and_validation_duration_seconds` | histogram | Specifies a number of assets that the service received for processing and validated within a given time series. |
+| `cms_services_handle_mutation_status_code` | counter | Specifies a number of `200` and `304` HTTP response status codes in a given time series returned by the mutation handler. |
```

Comment:
Please do not specify that there are only 200s and 304s, there can be more, unfortunately counter does not show you all values upfront with 0

We can omit this info

Reply:
and in other places same thing too

Reply:
yeah, I thought about consistency with other docs and the fact that there are 5 possible status codes in metadata service (afaik), but if we are documenting that there are only 2 possible values here: https://kyma-project.io/docs/master/components/asset-store/#details-supported-webhooks-service-specification-requirements, then I think it can stay

Reply:
After consideration, I will change to: `Specifies a number of different HTTP response status codes in a given time series. `